setup.py

```diff
@@ -24,6 +24,25 @@
     long_description_content_type="text/markdown",
     url="https://github.com/SymbiFlow/prjxray",
     packages=['prjxray'],
+    install_requires=[
+        'sdf-timing @ git+https://github.com/symbiflow/python-sdf-timing',
+        'fasm @ git+https://github.com/symbiflow/fasm',
```

Comment:
Don't use an `@` in `setup.py`.  If you need to pull in a git package, do it via `requirements.txt`.

Reply:
the problem this PR solves is that those were not installed when we do `pip install git+https://github.com/symbiflow/prjxray` (and we do that in symbiflow-examples). That is why I put those `@` in `setup.py`

Reply:
Never put `@` in setup.py, end of story.  It causes all kinds of problems.  `@` is an extremely narrow pin, and causes problems.  The longer term solution is for these packages to end up on PyPi, etc, but that is a longer term problem.

Reply:
> `@` is an extremely narrow pin

Can you explain why @litghost? Does it tie a dependency like `fasm` to the revision which is currently `master` on `fasm` forever if I install a package with `fasm @ git+https://github.com/symbiflow/fasm` in `install_requires`?

Reply:
The `@` is effectively "always use exactly this dependency".  The new pip dependency resolver respects this directly, but it means that it mixes poorly with other configurations.  As a concrete example, if you want to use `git submodule` to pin fasm to a particular commit (for example in symbiflow-examples), that will conflict with the `@` here.

The bottom line is `@` can **never** be used in `setup.py`, period.  There are no acceptable reasons to do so.

Reply:
`fasm` is now on PyPi (per https://github.com/SymbiFlow/fasm/pull/59).  If someone can do the same work for sdf-timing, the `@` can go away from this completely!